webapp/src/app/user-data.service.ts

```diff
@@ -29,6 +31,19 @@ export class UserData extends Data {
         const config = Config.fromTOML(await res.text());
         this.bc = await ByzCoinRPC.fromByzcoin(config.roster, config.byzCoinID);
     }
+
+    async load() {
```

Comment:
What's the use? `d` is not returned

Reply:
That's the old style loading that is still needed with the data-service, as you load it before the system has the possibility of fetching the configuration.